fitsio-sys/build.rs

```diff
@@ -33,3 +34,91 @@ PKG_CONFIG_PATH=<blah> cargo build
         Err(e) => panic!("Unhandled error: {:?}", e),
     };
 }
+
+fn main() {
+    #[cfg(not(feature = "static"))]
+    bind_cfitsio();
+
+    #[cfg(feature = "static")]
```

Comment:
Can we break this block out  to it's own function. Maybe have `main` call `bind_cfitsio`, and the implementation of `bind_cfitsio` change depending on the feature flag?